src/typed-ast/main.cc

```diff
@@ -0,0 +1,31 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+#include "ast/cli.h"
+#include "ast/module.h"
+#include "ast/pass.h"
+#include "ast/prec.h"
+#include "ast/ref.h"
+#include "ast/sugar.h"
+#include "ast/sym.h"
+#include "typed-ast/conversion.h"
+#include "typed-ast/print.h"
+
+int main(int argc, char** argv)
+{
+  pass::Passes passes = {{"sugar", sugar::build},
+                         {"sym", sym::build},
+                         {"ref", ref::build},
+                         {"prec", prec::build}};
+  err::Errors err;
+
+  auto opt = cli::parse(argc, argv);
+  auto m =
+    module::build(opt.grammar, opt.stopAt, passes, opt.filename, "verona", err);
+  auto typedModule = verona::ast::convertModule(m->ast);
```

Comment:
Are you planning to add a set of passes on top of your converted module?

So it becomes:
```
old ast -> sugar -> sym -> ref -> prec;
new ast = convert(prec) -> return;
```
then
```
old ast -> sugar -> sym -> ref;
new ast = convert(ref);
new ast -> new prec -> return;
```
then
```
old ast -> sugar -> sym;
new ast = convert(sym);
new ast -> new ref -> new prec -> return;
```
etc.

Reply:
I don't know yet. I don't have any immediate plans to, but it is bound to happen at some point.

Reply:
Makes sense, one thing at a time.